example/src/App.tsx

```diff
@@ -6,14 +6,15 @@ import Chargebee, { Product } from '@chargebee/react-native-chargebee';
 export default function App() {
   const site = 'site';
   const apiKey = 'apiKey';
-  const sdkKey = 'sdkKey';
+  const androidSdkKey = 'androidSdkKey';
+  const iOsSdkKey = 'iOsSdkKey';
 
   let productIdentifiers: string[] = [];
   let products: Product[] = [];
 
   useEffect(() => {
-    configure(site, apiKey, sdkKey);
-  }, []);
+    configure(site, apiKey, androidSdkKey, iOsSdkKey);
+  });
```

Comment:
Any reason why we're removing the empty dependency array here?

Reply:
To keep the linter happy :) 
We will be updating this example anyway.